indico/modules/rb_new/client/js/modules/calendar/serializers.js

```diff
@@ -15,7 +15,10 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
+import {ajax as roomSearchAjaxRules} from '../../common/roomSearch/serializers';
+
 export const ajax = {
+    ...roomSearchAjaxRules,
     start_date: ({date}) => date,
     end_date: ({date}) => date,
```

Comment:
i thought we don't have start/end date anymore in the calendar but only a single date

Reply:
Actually I don't think any date is needed there after all. These are serializers for the filters used in the searchRooms api and the day is used afterwards in the calendar api.